### DISCUSSION TITLE
Question: Which method does podman official recommend, root or rootless?

### DISCUSSION THREAD

**Discussion by loynoir** [containers/podman#14994] [General]
Given
- https://wiki.archlinux.org/title/Podman#Rootless_Podman

Question
- Which method does podman official recommend, root or rootless?

**Comment by rhatdan** [comment]
We recommend rootless if user namespace is available.  We believe that warning is fairly bogus and has been answered by the kernel developers who work on User Namespace.  Since most Distros (Ubuntu, Debian, Fedora, Centos, RHEL) default to supporting rootless mode we recommend that you use it.

**Reply by mheon** [reply]
If you need to use Root Podman, we do recommend using `--userns=auto` if possible to ensure your containers are not running as root, even if Podman itself is.
